src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/HttpClientTransport.cs

```diff
@@ -100,53 +102,24 @@ public HttpRequestMessage BuildRequestMessage()
                 }
 
                 if (_requestContent != null) {
-                    message.Content = new PipelineContentAdapter(_requestContent, Cancellation);
+                    message.Content = new PipelineContentAdapter(_requestContent, cancellation);
                     if (_contentTypeHeaderValue != null) message.Content.Headers.Add("Content-Type", _contentTypeHeaderValue);
```

Comment:
`Content-Type` and `Content-Length` are both members of [`HttpHeader`](https://github.com/Azure/azure-sdk-for-net/blob/3a24d924dc1cdf396601cae7c17e037c19897aab/src/SDKs/Azure.Base/data-plane/Azure.Base/Http/HttpHeader.cs).  You may want to consider using them over raw strings.

Reply:
I would prefer to limit the PR changes to its original intent. I'll go through header usages and clean them up after https://github.com/Azure/azure-sdk-for-net/pull/5545 goes in so I get fewer merge conflicts.